lib/helpers/config-helper.js

```diff
@@ -176,10 +178,11 @@ var api = {
     try {
       var urlParts = url.parse(urlString);
       var result   = {
-        database: urlParts.path.replace(/^\//,  ''),
+        database: urlParts.pathname.replace(/^\//,  ''),
```

Comment:
@sushantdhiman [this](https://github.com/sequelize/cli/pull/461/files#diff-072bd6704a4a833da23b826bb978aaf7R208) is the fix you added in sequelize, should work now right?